image/image.go

```diff
@@ -26,6 +26,9 @@ import (
 // validates the manifest pointed to by the given ref
 // or returns an error if the validation failed.
 func ValidateLayout(src, ref string) error {
+	if _, err := os.Stat(src); err != nil {
```

Comment:
This is unnecessary (and slightly racy) given the validate->pathwalker code does appropriate error handling anyway.